lib/iris/tests/unit/lazy_data/test_is_lazy_data.py

```diff
@@ -26,13 +26,13 @@
 import numpy as np
 import dask.array as da
 
-from iris._lazy_data import is_lazy_data
+from iris._lazy_data import as_lazy_data, is_lazy_data, _MAX_CHUNK_SIZE
 
 
 class Test_is_lazy_data(tests.IrisTest):
     def test_lazy(self):
-        lazy_values = np.arange(30).reshape((2, 5, 3))
-        lazy_array = da.from_array(lazy_values, 1e6)
+        values = np.arange(30).reshape((2, 5, 3))
+        lazy_array = da.from_array(values, _MAX_CHUNK_SIZE)
```

Comment:
Minor, but it might be worth using the kwarg i.e.
`lazy_array = da.from_array(values, chunks=_MAX_CHUNK_SIZE)`
to be consistent with the other times da.from_array is called.